DTML.EduBot/Dialogs/GreetingDialog.cs

```diff
@@ -14,14 +14,13 @@ public partial class RootDialog : QnaLuisDialog<object>
         [LuisIntent("Greeting")]
         public async Task Greeting(IDialogContext context, LuisResult result)
         {
-            string botresponse = BotPersonality.GetRandomGreeting(); 
-            await context.PostAsync(botresponse);
+            await ConvertAndPostResponse(context, BotPersonality.GetRandomGreeting());
```

Comment:
Let's confirm with @soumyavijay if we want to translate everything. I think the idea is to keep translations to minimum. Instead translate when explicitly asked by the user - "Can you translate XXXX to english/ How do you say XXXX in english"